### PR TITLE
Backport 91614390 grunt angular templates

### PR DESCRIPTION
**Pull request by mattc-cogapp**
The bulk of the changes can be found in the commit message for https://github.com/Southbank-Centre/SC-app/commit/15480259ea882b149b37e960cff51059a38834bc

### CODE REVIEW

**Comment by mattc-cogapp** [comment]
Sorry @samueljmurray, I assigned this before I'd completed the testing...I forgot a couple of bits.

Please ignore for now, and I'll re-assign it when I'm done!

**Comment by samueljmurray** [comment]
Thanks @mattc-cogapp I'll wait for your next update

**Comment by mattc-cogapp** [comment]
Hi @samueljmurray, sorry for the delay with this...we ran the entire browser testing tests just to be sure. This should now be ready for review!

File: .htaccess [hunk 1]
```diff
@@ -259,12 +259,12 @@ AddDefaultCharset utf-8
 # | Route /event* and /events* to /whats-on |
 # -------------------------------------------
 
-<IfModule mod_rewrite.c>
-    RewriteBase /
-    RewriteRule ^(#/events|#/event) /#/whats-on [NE,R]
-    RewriteRule ^(#events|#event) /#/whats-on [NE,R]
-    RewriteRule ^(events|event) /whats-on [NE,R,L]
-</IfModule>
+#<IfModule mod_rewrite.c>
```
Comment (samueljmurray) [inline .htaccess]: @mattc-cogapp how come this is commented out?

Comment (mattc-cogapp) [inline .htaccess]: Events needed to live under /events. for Web We Want This was fine when navigating to the page through menus etc, but when you refresh the page the URL rewriting kicks in and you get redirected to a 404 as there's no route for /whats-on any more.

Comment (samueljmurray) [inline .htaccess]: Cool, makes sense

**Comment by samueljmurray** [comment]
@mattc-cogapp all looks good. You'll see a couple of commits from me - I changed the 'festival' hash to be called 'appDetails' because not all apps are for festivals. Could you double check for me that I've changed it's name in all of the places needed. Let me know and I'll do the merge if you're happy

**Comment by mattc-cogapp** [comment]
Yeah, that looks fine and makes sense! Thanks!

**Pull request merged**